crates/feldera-storage/src/lib.rs

```diff
@@ -0,0 +1,25 @@
+/// The storage backend.
+///
+/// TODO: This is currently only `pub` because we use it in the bench binary.
```

Comment:
I found a way to make it not pub by calling the binary main.rs in the project root instead of putting it under bin/binary.rs.. need to remove this comment